### PR TITLE
CCDScan documentation

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

The documentation for CCDScan was not made by Fintech Builders and we need to provide something for our users.

## Changes

Added sections to the existing CCDScan topic and took screenshots to explain what the users are seeing and can do in CCDScan.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

File: source/mainnet/net/resources/ccd-scan.rst [hunk 1]
```diff
@@ -21,3 +21,234 @@ Delegation and baker pools
...
+Below the graphs, the table contains the following:
+
+- **Block hash**: the first six digits of the block hash. You can click Copy |copy| to copy the entire hash.
+- **Status**: the block status
+- **Height**: block height
+- **Age**: block age
+- **Baker**: the baker ID of the baker who baked the block
+- **Transactions**: the number of transactions in the block
+
+Remember that this information is dependent on whether you have selected Mainnet or Testnet.
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/resources/ccd-scan.rst:175]: Moved to the point about the selector on the home screen and removed other places.

File: source/mainnet/net/guides/baker-pool.rst [hunk 2]
```diff
@@ -30,9 +30,9 @@ Research a baker pool
 
 Before delegating stake to a baker pool, it is important to research the pool and get an idea of performance and how it is managed.
 
-The first thing to check before delegating stake to a baker pool is the URL for the baker pool. This is information that the baker can provide about the pool. It is not required that the baker provide this, but recommended. You can find the URLs on :ref:`CCD Scan<ccd-scan>`.
+The first thing to check before delegating stake to a baker pool is the URL for the baker pool. This is information that the baker can provide about the pool. It is not required that the baker provide this, but recommended. You can find the URLs on :ref:`CCD Scan<bakers-view>`.
 
-Make sure that the pool isn't close to having too many delegators or close to max capital.Another indicator of good management is reliability; check the uptime of the node.
+Make sure that the pool isn't close to having too many delegators or close to max capital. Another indicator of good management is reliability; check the :ref:`uptime of the node<ccdscan-nodes>`.
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/guides/baker-pool.rst:35]: Fair enough. What would be a good indicator of reliability?

[the remark this answers is not in the document]
Comment (abizjak) [reply source/mainnet/net/guides/baker-pool.rst]: The best one would be the ratio between expected blocks and produced blocks in a time period (e.g., payday), and if the pool is a finalizer, the ratio between the number of finalizations and the number of finalization records the pool signed.

I can't find this summary presented on the CCDScan though.

File: source/mainnet/net/resources/ccd-scan-staking.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+The graphs show when the next payday happens, the amount of staked CCD, the number of bakers, and the total of rewards during the selected time range in the filter. The **Previous payday block** contains information about all rewards paid out for a payday period. If you click **Previous payday block**, a special block details screen for the previous pay day block appears.
+
+.. image:: ../images/ccd-scan/ccd-scan-staking-payday-block-details.png
+
+|
+
+The Pay Day block details contains the following:
+
+- **Block**: the first six digits of the block hash. Click Copy |copy| to copy the entire hash. The block status is also shown.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-staking.rst:21]: digits -> characters

File: source/mainnet/net/resources/ccd-scan-staking.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- **Age**: the age of the block and date/time stamp when it was created.
+- **Baker id**: ID of the baker who baked the block. Click the baker IDto see the :ref:`baker details<home-screen-baker>`.
+- **Tokenomics**: contains the following details about the block:
+    - Distributed minted CCD: Contains the total minted CCDs and a breakdown on their distribution across the baker reward account, finalization reward account, and foundation account.
+    - Accrued block rewards: similar to a ‘regular’ :ref:`block<home-screen-block>`; contains the accrued block rewards.
+    - Payday foundation rewards: displays the payout to the foundation account for the pay day period and contains the option to drill-through to the :ref:`foundation account<accounts-view>`.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-staking.rst:27]:
```suggestion
    - Payday foundation rewards: displays the payout to the foundation account for the payday period and contains the option to drill-through to the :ref:`foundation account<accounts-view>`.
```
Should we use `payday` everywhere?

File: source/mainnet/net/resources/ccd-scan-staking.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- **Delegators APY (30 days)**: the calculated yield of the delegators for the past 30 days.
+- **Delegation pool Status**: the baker's pool status.
+- **Delegators**: the number of delegators that baker has if the baker has an open pool
+- **Available for delegation**: shows the maximum number of remaining CCDs available for delegation based on the capital bound and the leverage factor between a baker and associated delegators. Based on the :ref:`cap<delegation-caps>` imposed.
+- **Total stake**:  shows the combined stake of the baker and delegators (if any) and the percentage breakdown of the split between the two.
+
+Top delegators view
+===================
+
+If you select **Top delegaators** you see a list of all delegators (sorted by their staked amount. With the ability to see account details plus inspect details about the delegation target (baker pool or passive delegation), and so on. The table contains the following:
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-staking.rst:51]:
```suggestion
If you select **Top delegators** you see a list of all delegators (sorted by their staked amount. With the ability to see account details plus inspect details about the delegation target (baker pool or passive delegation), and so on. The table contains the following:
```

Comment (abizjak) [reply source/mainnet/net/resources/ccd-scan-staking.rst:51]: Also something seems wrong with parenthesis in this sentence.

File: source/mainnet/net/resources/ccd-scan-staking.rst [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+If you select **Passive delegation**  it is possible to see the size of the passive delegation in total and information about the yield from delegating to passive delegation. Furthermore it is possible to see the commission rates for passive delegation.
+
+.. image:: ../images/ccd-scan/ccd-scan-passive-delegation.png
+
+- **Delegated stake**: amount of CCD delegated to passive delegation.
+- **APY (7 days)**: the calculated yield of passive delegation based on past 7 days.
+- **APY (30 days)**: the calculated yield of passive delegation based on past 30 days.
+- **Baking commission**: percent of rewards paid as commission to baker.
+- **Finalisation commission**: percent of finalization commission.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-staking.rst:73]: percent of finalization rewards that are paid as commission to the baker

File: source/mainnet/net/resources/ccd-scan-staking.rst [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+- **Transaction commission**: the transaction commission percent.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-staking.rst:74]: percentage of transaction rewards that are paid as commission to the baker

File: source/mainnet/net/resources/ccd-scan-staking.rst [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+.. image:: ../images/ccd-scan/ccd-scan-baker-details.png
+
+|
+
+The baker details shows the following information about the baker.
+
+- **Baker**: the baker ID. This is the ID you need if you want to delegate to the baker pool in a wallet. If the Baker has been removed (i.e. the baker is no longer active), **Removed** appears next to the ID and a timestamp shows when the baker stopped.
+- If there are any pending changes, they are shown here. Events such as stake increase, stake reduction, or baker stop are displayed.
+- **Total stake**: the baker's total stake. Below the total is the amount staked by the baker and the amount of stake that comes from delegators.
+- **Open, Closed**: the baker pool status.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-staking.rst:97]: Perhaps elaborate on this. Open means you can delegate to it, closed means you cannot.

File: source/mainnet/net/resources/ccd-scan-staking.rst [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+- **Account**: the first six characters of the baker account. You can click on the characters for more :ref:`information about the sender<home-screen-sender>`. You can click Copy |copy| to copy the entire account number.
+- **APY Period**: choose between 7 days or 30 days. This seletion affects the **Total APY**, **Baker APY**, and **Delegators APY**.
+- **Baker rank**: the baker's rank out of all bakers on Mainnet/Testnet based on the baker's total stake.
+- **Baking commission**: the baker's baking commission percent.
+- **Finalisation commission**: the baker's finalization commission percent.
+- **Transaction commission**: the baker's transaction commission percent.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-staking.rst:103]: I would rephrase as above for passive delegationl.

File: source/mainnet/net/resources/ccd-scan-transactions.rst [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+|
+
+The transaction details shows the following.
+
+- **Transaction**: the first six characters of the transaction hash. Click Copy |copy| to copy the entire transaction hash. It also shows the transaction state.
+- **Block height/block hash**: the block height and the first six characters of the block hash. Click Copy |copy| to copy the entire block hash. Click the block hash to see the :ref:`account details<blocks-view>`.
+- **Age**: the transaction age with date/time stamp.
+- **Transaction type/cost**: the transaction type and the transaction fee.
+- **Sender**: the first six characters of the account that sent the transaction if available for that transaction. Click Copy |copy| to copy the entire hash. Click the account hash to see the :ref:`account details<accounts-view>`.
+- **Events/Reject reason**:  list of events included in the transaction. It contains more details about the transaction, including drill-through links and copy options where available. There are many different transaction types and descriptions. If you see the transaction type **Update micro CCD per Euro**, it is possible to see the new rate and the effective timestamp.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-transactions.rst:43]: Is "Update micro CCD per Euro" treated in a special way? There are other chain updates, are those not displayed in detail?

File: source/mainnet/net/resources/ccd-scan.rst [hunk 11]
```diff
@@ -20,4 +20,56 @@ CCDScan
...
+.. image:: ../images/ccd-scan/ccd-scan-overview.png
+
+|
+
+1.  Mainnet/Testnet selector. The information shown is dependent on whether you have selected Mainnet or Testnet.
+2.  Search field: search across all relevant blockchain data for specific information, such as a particular account, transaction, block, or baker and see all relevant details associated with that entity.
+3.  Site naviation: change between pages to see other information. On a smaller screen, the page selector is available in the hamburger menu |hamburger|.
+4.  Time range filter: controls the time range of the data shown.
+5.  Total CCD in existence on either Mainnet or Testnet; on Mainnet Total CCD in circulation is also shown.
+6.  Blocks added: sum of blocks added in the selected time range.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan.rst:44]: sum? You mean number of blocks added?

File: source/mainnet/net/resources/ccd-scan-blocks.rst [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+.. image:: ../images/ccd-scan/ccd-scan-blocks.png
+
+|
+
+The graphs show the number of blocks added, the average block time, and the average finalization time.
+
+Below the graphs, the table contains the latest blocks added to the blockchain with the following information:
+
+- **Block hash**: the first six digits of the block hash. Click Copy |copy| to copy the entire hash. Click the block hash to see :ref:`block details<home-screen-block>`.
+- **Status**: the block status
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/resources/ccd-scan-blocks.rst:18]: I have not seen that you can see blocks in that status before they are finalized. Only Finalized or Rejected.

File: source/mainnet/net/resources/ccd-scan-blocks.rst [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+- **Height**: block height
+- **Age**: block age
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/resources/ccd-scan-blocks.rst:20]: Yes.

File: source/mainnet/net/resources/ccd-scan-staking.rst (hunk 4 above)
Comment (dg-concordium) [reply source/mainnet/net/resources/ccd-scan-staking.rst:27]: We have used pay day everywhere else in the documentation. However, on the CCDScan screen it is written as payday so I have to write it as readers see it.

File: source/mainnet/net/resources/ccd-scan-blocks.rst [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+
+|
+
+The block details shows the following information about the block.
+
+- **Block**: the first six characters of the block hash. You can click Copy |copy| to copy the entire hash.
+- **Finalised** or **Rejected**: state of the block.
+- **Age**: age of the block with date/time stamp.
+- **Baker id**: baker ID of the baker who baked the block. Click on the baker ID for details about the :ref:`baker<home-screen-baker>`.
+- **Tokenomics**: shows the tokenomics of the accrued block rewards, including the minted CCDs for the block with a breakdown of the distribution plus finalisers and baker rewards. A subsection of tokenomics is the full list of finalisers for each individual block including their weight and reward. It is possible to copy the full address of the finalizer |copy| or drillthrough to the account information of the finaliser. Baker rewards includes a breakdown of the baker rewards details about the rewards, transaction fees, and so on.
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/resources/ccd-scan-blocks.rst:41]: Copy paste error.

File: source/mainnet/net/guides/baker-pool.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/guides/baker-pool.rst]: https://github.com/Concordium/concordium-scan/issues/11

File: source/mainnet/net/resources/ccd-scan-staking.rst (hunk 4 above)
Comment (dg-concordium) [reply source/mainnet/net/resources/ccd-scan-staking.rst:27]: https://github.com/Concordium/concordium-scan/issues/2

File: source/mainnet/net/resources/ccd-scan-blocks.rst (hunk 12 above)
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/resources/ccd-scan-blocks.rst]: I have just confirmed with Emil K. that there is no in between status for baked but not finalized blocks.

File: source/mainnet/net/resources/ccd-scan-blocks.rst [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+When you click a block hash, the following appears:
+
+.. image:: ../images/ccd-scan/ccd-scan-home-block-hash.png
+
+|
+
+The block details shows the following information about the block.
+
+- **Block**: the first six characters of the block hash. You can click Copy |copy| to copy the entire hash.
+- **Finalised** or **Rejected**: state of the block.
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/resources/ccd-scan-blocks.rst:38]: Confirmed with Emil K. that Rejected has been removed from the view.

File: source/mainnet/net/resources/ccd-scan-accounts.rst [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+- **Age**: the age of the account including the exact timestamp for all individual accounts.
+- **Transactions**: a list of all transactions associated with the account, including incoming and outgoing transfers. Use the navigation buttons at the bottom to navigate through transactions.
+- **Release schedule**: shown only for accounts that have transfers with a release schedule to show the remainder of the schedule (non-released assets). The list of release dates and amounts is available, including a drill-through link to the :ref:`underlying transaction(s)<home-screen-transaction>`.
+- **Account statement**: shows how the account balance has changed over time, including the types of changes that have impacted the balance of the account. Use the navigation buttons at the bottom to navigate through account statements.
+- **Baker**: if an account is also a baker, the baker ID and the staked amount is shown here. Click the baker ID to see :ref:`baker details<home-screen-baker>`.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-accounts.rst:46]:
```suggestion
- **Baker**: if an account is also a baker, the baker ID and the staked amount are shown here. Click the baker ID to see :ref:`baker details<home-screen-baker>`.
```
?

File: source/mainnet/net/resources/ccd-scan-nodes.rst [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+|
+
+- **Node**: shows the name of the node
+- **Baker**: baker ID of the node if it is a baker. Click the baker ID to see the :ref:`baker details<home-screen-baker>`.
+- **Uptime**: amount of time the node has been running.
+- **Client version**: version of concordium node software the node is running.
+- **Average ping**: average network delay between the node and its peers.
+- **Packets sent**: number of packets sent by the participant during uptime. Packets are transactions, blocks, finalization messages.
+- **Packets received**: number of packets received by the participant during uptime. Packets are transactions, blocks, finalization messages.
+- **Baking committee**: Whether the node is currently running as a baker. Will be either **Active member** or **Not a member**.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-nodes.rst:41]: Is this meant to be capitalized? The other lines are not.

File: source/mainnet/net/resources/ccd-scan-nodes.rst [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+- **Best block statistics**: shows statistics about the best block.  Click Copy |copy| to copy the entire block hash. Click the block hash to see :ref:`block details<home-screen-block>`.
+- **Last finalized block statistics**: shows statistics about the last finalized block in participant’s chain.  Click Copy |copy| to copy the entire block hash. Click the block hash to see :ref:`block details<home-screen-block>`.
+- **Peers**: Shows the current list of peers of the node with the ability to drill-through and inspect peer node information.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-nodes.rst:44]: Also this.

File: source/mainnet/net/resources/ccd-scan-transactions.rst [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+The transaction details shows the following.
+
+- **Transaction**: the first six characters of the transaction hash. Click Copy |copy| to copy the entire transaction hash. It also shows the transaction state.
+- **Block height/block hash**: the block height and the first six characters of the block hash. Click Copy |copy| to copy the entire block hash. Click the block hash to see the :ref:`account details<blocks-view>`.
+- **Age**: the transaction age with date/time stamp.
+- **Transaction type/cost**: the transaction type and the transaction fee.
+- **Sender**: the first six characters of the account that sent the transaction if available for that transaction. Click Copy |copy| to copy the entire hash. Click the account hash to see the :ref:`account details<accounts-view>`.
+- **Events/Reject reason**:  list of events included in the transaction. It contains more details about the transaction, including drill-through links and copy options where available. There are many different transaction types and descriptions.
+
+A successful transaction shows the **Events** that were included in the transaction.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-transactions.rst:45]:
```suggestion
A successful transaction shows the **Events** that were the result of the transaction.
```

File: source/mainnet/net/resources/ccd-scan-transactions.rst [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+
+If the transaction was rejected, you see something similar to the following:
+
+.. image:: ../images/ccd-scan/ccd-scan-home-transaction-reject.png
+
+|
+
+The only difference between a successful and rejected transaction is that the rejected transaction shows the **Reject reason** instead of the events.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan-transactions.rst:53]: Not sure what the context for "the only difference" is.

But I would not phrase it like this. Instead I would say something like

"The only effect of a rejected transaction is the payment for execution cost. In contrast, a successful transaction affects the state of the chain. The effects are described by the generated events."

File: source/mainnet/net/resources/ccd-scan.rst [hunk 21]
```diff
@@ -20,4 +20,56 @@ CCDScan
...
+
+On the Home page that appears when you navigate to CCDScan, you see the following information:
+
+.. image:: ../images/ccd-scan/ccd-scan-overview.png
+
+|
+
+1.  Mainnet/Testnet selector. The information shown is dependent on whether you have selected Mainnet or Testnet.
+2.  Search field: search across all relevant blockchain data for specific information, such as a particular account, transaction, block, or baker and see all relevant details associated with that entity.
+3.  Site naviation: change between pages to see other information. On a smaller screen, the page selector is available in the hamburger menu |hamburger|.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan.rst:41]:
```suggestion
3.  Site navigation: change between pages to see other information. On a smaller screen, the page selector is available in the hamburger menu |hamburger|.
```

File: source/mainnet/net/resources/ccd-scan.rst [hunk 22]
```diff
@@ -20,4 +20,56 @@ CCDScan
...
+4.  Time range filter: controls the time range of the data shown.
+5.  Total CCD in existence on either Mainnet or Testnet; on Mainnet Total CCD in circulation is also shown.
+6.  Blocks added: number of blocks added in the selected time range.
+7.  Block Time: average time since last block created in the selected time range.
+8.  Transactions: sum of transactions in the selected time range.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan.rst:46]: What does sum mean? Total number of transactions in the selected time range? I suggest rephrasing to make clearer.

File: source/mainnet/net/resources/ccd-scan.rst [hunk 23]
```diff
@@ -20,4 +20,56 @@ CCDScan
...
+9.  Accounts created: sum of accounts created in the selected time range.
```
Comment (abizjak) [inline source/mainnet/net/resources/ccd-scan.rst:47]: Same here.

**Review by abizjak — APPROVED** [review]
